public/app/features/explore/LiveLogs.tsx

```diff
@@ -82,24 +152,33 @@ class LiveLogs extends PureComponent<Props> {
           <div
             ref={element => {
               this.liveEndDiv = element;
-              if (this.liveEndDiv) {
+              // This is triggered on every update so on every new row. It keeps the view scrolled at the bottom by
+              // default.
+              if (this.liveEndDiv && !isPaused) {
                 this.liveEndDiv.scrollIntoView(false);
               }
             }}
           />
         </div>
         <div className={cx([styles.logsRowsIndicator])}>
-          <span>
-            Last line received: <ElapsedTime resetKey={this.props.logsResult} humanize={true} /> ago
-          </span>
-          <LinkButton
-            onClick={this.props.stopLive}
-            size="md"
-            variant="transparent"
-            style={{ color: theme.colors.orange }}
+          <button
```

Comment:
Using the old buttons here as with new one the inverse looked like this:
<img width="362" alt="Screenshot 2019-09-02 at 14 20 42" src="https://user-images.githubusercontent.com/1014802/64117680-19042000-cd96-11e9-9d0c-ac216cdb1c4f.png">

Also there are some issues with the icons, which I will make a PR for. In general not sure if the old styles should be ported to new buttons or if there is some design work still being done. @Ijin08 @dprokop